pkg/noderesourcetopology/pluginhelpers.go

```diff
@@ -139,12 +144,41 @@ func makePodByResourceListWithManyContainers(resources *v1.ResourceList, contain
 func extractResources(zone topologyv1alpha1.Zone) v1.ResourceList {
 	res := make(v1.ResourceList)
 	for _, resInfo := range zone.Resources {
-		klog.V(5).InfoS("Extract resources for zone", "resName", resInfo.Name, "resAvailable", resInfo.Available)
 		res[v1.ResourceName(resInfo.Name)] = resInfo.Available
 	}
 	return res
 }
 
+func logNumaNodes(desc string, nodes NUMANodeList) {
+	if !klog.V(6).Enabled() {
+		// avoid even the trivial range over nodes
+		return
+	}
+	for _, numaNode := range nodes {
+		numaIdent := fmt.Sprintf("node-%d", numaNode.NUMAID)
+		klog.V(6).InfoS(desc, resourceListToLoggable(numaIdent, numaNode.Resources)...)
+	}
+}
+
+func resourceListToLoggable(ident string, resources v1.ResourceList) []interface{} {
```

Comment:
Can we have some unit-tests here?
It can also be useful getting some sense of the format this function return.

Reply:
yes, totally. I'll need to add examples and some tests coverage.